prime-router/src/main/kotlin/azure/HistoryFunctions.kt

```diff
@@ -334,74 +334,107 @@ open class BaseHistoryFunction : Logging {
         reportIdIn: String,
         context: ExecutionContext
     ): HttpResponseMessage {
-        val authClaims = checkAuthenticated(request, context)
```

Comment:
Let's just double check this auth update will not break this endpoint for certain users. For example, was this working for the *.reports scope before?

Reply:
I'll confirm, but also might just add just to be safe.

Reply:
FYI, [this](https://github.com/CDCgov/prime-reportstream/blob/a019095ebf334ad67cdd860bffe8130fe236646b/prime-router/src/main/kotlin/azure/HistoryFunctions.kt#L541) is where the check was previously looking and actually seemed to only accept being part of the org exactly; i.e.
`DHSender_xyz` would be allowed in but `DHSender_xyzAdmins` would not

Reply:
Added *.*reports just in case